src/Network/AWS/S3/StreamingUpload.hs

```diff
@@ -7,38 +7,40 @@
 {-# LANGUAGE TupleSections #-}
 
 
-module Network.AWS.S3.StreamingUpload
+module Network.AWS.S3.StreamingUpload -- TODO: Rename to Amazonka
```

Comment:
Leaving this to the end because I want to make rebasing from `rewrite-play` as easy as possible.